docs/README.rst

```diff
@@ -1,49 +0,0 @@
-###################
```

Comment:
I added one that just points to index.rst.  I am trying to minimize duplication.